library/core/src/panic/panic_info.rs

```diff
@@ -62,6 +65,12 @@ impl<'a> PanicInfo<'a> {
         self.payload = info;
     }
 
+    #[unstable(feature = "error_in_panic", issue = "none")]
```

Comment:
Can you [open a tracking issue](https://github.com/rust-lang/rust/issues/new?assignees=&labels=C-tracking-issue%2C+T-libs&template=library_tracking_issue.md&title=Tracking+Issue+for+XXX) for this feature? Thanks!

Reply:
Tracking issue is here: https://github.com/rust-lang/rust/issues/103820